lib/vast/Conversion/Common/CoreToLLVM.cpp

```diff
@@ -465,6 +465,25 @@ namespace vast
         using sub = one_to_one< hl::SubIOp, LLVM::SubOp >;
         using mul = one_to_one< hl::MulIOp, LLVM::MulOp >;
 
+        using fadd = one_to_one< hl::AddFOp, LLVM::FAddOp >;
+        using fsub = one_to_one< hl::SubFOp, LLVM::FSubOp >;
+        using fmul = one_to_one< hl::MulFOp, LLVM::FMulOp >;
+
+        using sdiv = one_to_one< hl::DivSOp, LLVM::SDivOp >;
+        using udiv = one_to_one< hl::DivUOp, LLVM::UDivOp >;
+        using fdiv = one_to_one< hl::DivFOp, LLVM::FDivOp >;
+
+        using srem = one_to_one< hl::RemSOp, LLVM::SRemOp >;
+        using urem = one_to_one< hl::RemUOp, LLVM::URemOp >;
+        using frem = one_to_one< hl::RemFOp, LLVM::FRemOp >;
+
+        using bor = one_to_one< hl::BinOrOp, LLVM::OrOp >;
+        using band = one_to_one< hl::BinAndOp, LLVM::AndOp >;
+        using bxor = one_to_one< hl::BinXorOp, LLVM::XOrOp >;
+
+        using shl = one_to_one< hl::BinShlOp, LLVM::ShlOp >;
+        using shr = one_to_one< hl::BinShlOp, LLVM::ShlOp >;
```

Comment:
We may want to use type list to aggregate them (so later just the type list can be added to patterns) and instantiate `one_to_one`.